Library/Homebrew/rubocops/cask/url.rb

```diff
@@ -24,19 +24,35 @@ class Url < Base
         def on_url_stanza(stanza)
           return if stanza.stanza_node.block_type?
 
+          url_stanza = stanza.stanza_node.first_argument
           hash_node = stanza.stanza_node.last_argument
           return unless hash_node.hash_type?
 
           hash_node.each_pair do |key_node, value_node|
             next unless key_node.source == "verified"
             next unless value_node.str_type?
-            next unless value_node.source.start_with?(%r{^"https?://})
+
+            if value_node.source.start_with?(%r{^"https?://})
+              add_offense(
+                value_node.source_range,
+                message: "Verified URL parameter value should not contain a URL scheme.",
+              ) do |corrector|
+                corrector.replace(value_node.source_range, value_node.source.gsub(%r{^"https?://}, "\""))
+              end
+            end
+
+            # Skip if the URL and the verified value are the same.
+            next if value_node.source == url_stanza.source.gsub(%r{^"https?://}, "\"")
+            # Skip if the URL has two path components, eg: `https://github.com/google/fonts.git`.
+            next if url_stanza.source.gsub(%r{^"https?://}, "\"").count("/") == 2
```

Comment:
@issyl0, this only fixes this particular case.

What if this isn't a GitHub URL with this specific format?

```ruby
url "https://example.org/long/path/app-1.2.3.dmg",
    verified: "example.org/long/path/app"
```

Granted I don't think we currently have any cask with such a format, but `2` is kinda arbitrary.

Reply:
I think we are overcomplicating the condition for this cop.

If `verified` doesn't already end with a slash but the `url` starts with "#{verified}/", a slash should be added.